docs/content/how-to/creating-tables.md

```diff
@@ -415,25 +421,14 @@ val dataset = spark.read.format("paimon").load("hdfs://path/to/table")
 
 {{< tab "Hive" >}}
 
-* To access existing paimon table, you can also register them as external and internal tables in Hive. The following SQL creates an external table named `my_table`, where the base path of table files is `hdfs://path/to/table`. As schemas are stored in table files, users do not need to write column definitions.
+To access existing paimon table, you can also register them as external and internal tables in Hive. The following SQL creates an external table named `my_table`, where the base path of table files is `hdfs://path/to/table`. As schemas are stored in table files, users do not need to write column definitions.
```

Comment:
To access existing paimon table, you can also register them as external tables in Hive.